builder/googlecompute/config.go

```diff
@@ -195,6 +195,13 @@ type Config struct {
 	// The path to a startup script to run on the VM from which the image will
 	// be made.
 	StartupScriptFile string `mapstructure:"startup_script_file" required:"false"`
+	// If "true", the contents of `startup_script_file` is wrapped in a Packer specific
+	// script that tracks the execution and completion of the provided
+	// script file. The wrapper ensures that the builder will not continue until
+	// the startup script has been executed. Defaults to "true".
```

Comment:
May be worth mentioning in the docs that in the future we intend to make this "false"